app/models/work_version.rb

```diff
@@ -193,6 +193,8 @@ def purl_reservation?
   end
 
   def previous_version
-    work.work_versions.find { |check_work_version| check_work_version.version = version - 1 }
+    return nil if version == 1 # shortcut any query checks if we are on the first version
+
+    work.work_versions.find { |check_work_version| check_work_version.version == version - 1 }
   end
```

Comment:
we were not doing a conditional check before, but instead just subtracting one each time, so always returning something even if we were on version 1